VikingQuiz.UI/src/Components/MyQuizesPage/DisplayPlayedGamesComponent/DisplayPlayedGamesComponent.scss

```diff
@@ -0,0 +1,113 @@
+                justify-content: space-between;
+                .played-game-informations{
+                    width: 60%;
+                    margin: auto;                  
+                }
+                .played-game-photo{
+                    width: 40%;
+                    justify-content: center;
+                    .played-quiz-photo{
+                        margin-top: 15%;
+                        margin-left: auto;
+                        margin-right: auto;
+                        border-radius: 50%;
+                        width: 80%;
+                        height: 70%;                     
+                    }
+                }
+                .played-game-informations{
+                    margin-top: 0px;
+                    margin-bottom: 0px;
+                    margin-left: 0px;
+                    .number-of-players-container{
+                        display: inline-flex;
+                        width: 100%;
+                        left: 0px;                      
+                        .player-icon{
+                            width: 25px;
+                            height: 25px;
+                            background: transparent url(./../../../media/player_icon.png) no-repeat;
+                            background-size: 100%;
+                            margin-right: 4px;
+                            margin-left: 8%;
+                            margin-top: 20%;                           
+                        }
+                        .players-number{
+                            color: #0c4750;
+                            margin-top: 24%;
+                        }
+                    }
+                    .game-date-container{
+                        display: inline-flex;
+                        width: 100%;
+                        left: 0px;
+                        .calendar-icon{                          
+                            width: 25px;
+                            height: 25px;
+                            background: transparent url(./../../../media/calendar.png) no-repeat;
+                            background-size: 100%;
+                            margin-right: 4px;
+                            margin-left: 8%;
+                        }
+                        .game-date{
+                            color: #0c4750;
+                            margin-top: 4%;
+                        }
+                    }
+                }
+            }
+            .played-quiz-title{        
+                background-color: #0c4750;
+                color: white;
+                margin: 4px;
+                letter-spacing: 1px;
+            }           
+        }
+        @media screen and (min-width: 1300px){            
+            max-width: none;
+            .played-game-item{
+                min-width: none;
+                margin-right: 4%;
+            }     
+        }
+        @media screen and (min-width: 1400px){
+            max-width: none;
```

Comment:
max-width and min-width properties are redundant for the 1400px media queries because the same rules are already applied form the min-width: 1300px